client/lib/plugins/wporg-data/test/actions.js

```diff
@@ -43,6 +48,17 @@ describe( 'WPorg Data Actions', () => {
 		assert.equal( mockedWporg.getActivity().fetchPluginsList, 1 );
 	} );
 
+	it( 'should call fetchCuratedList for the "featured" category', () => {
```

Comment:
Maybe we can rephrase the two following tests to, `Should return our Calypso curated feature list` and `Does not return the community featured list`